InjectionByExample.Tests/ActivatorTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using InjectionByExample;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+
+namespace InjectionByExample.Tests
+{
+    [TestClass()]
+    public class ActivatorTests
+    {
+        IList<Registration> registrations = new List<Registration>()
+        {
+            new Registration(typeof(Driver), typeof(Driver), Lifetime.NewInstance),
+            new Registration(typeof(ICar), typeof(Car), Lifetime.InstancePerContainer),
+            new Registration(typeof(IEngine), typeof(Engine), Lifetime.SingleInstance)
+        };
+
+        Container container;
+
+        public ActivatorTests()
+        {
+            var containerBuilder = new ContainerBuilder();
+
+            containerBuilder.Register<Driver>();
+            containerBuilder.Register<ICar, Car>(Lifetime.InstancePerContainer);
+            containerBuilder.Register<IEngine, Engine>(Lifetime.SingleInstance);
+
+            container = containerBuilder.Build();
+        }
+
+        [TestMethod()]
```

Comment:
Unfortunately it is not clear to me what exactly you are verifying here

Reply:
I forgot to delete the constructor because I no longer needed it. Sorry. I think there is not much scope for testing in this case.